stack.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-resolver: nightly-2018-10-01
+resolver: lts-13.7
```

Comment:
This should probably be its own PR. However, if it compiles I'm happy keeping it.

Reply:
It compiles but I'll re-submit as a separate PR on Mon, you are right.